syntax error in go mod file for provider

There was an error intoroduced:
unos-terraform/cmd/processProviders$ go build
# github.com/Juniper/junos-terraform/Internal/processProviders
../../Internal/processProviders/go_mod.go:3:22: string not terminated

which makes the build fail.

Needs correcting.

--- a/Internal/processProviders/go_mod.go
+++ b/Internal/processProviders/go_mod.go
@@ -9,4 +9,4 @@ require (
         github.com/davedotdev/go-netconf v0.1.5
 	github.com/hashicorp/terraform-plugin-log v0.4.0
         github.com/hashicorp/terraform-plugin-sdk/v2 v2.16
-)
+)`

